README.md

```diff
@@ -10,10 +10,11 @@
 - Js, HTML, CSS
 - Webpack, Jest.
 - GIT, VS-code.
+- Linter
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://crystallinebutterfly.github.io/Group-Capstone-1/)
 
 
 ## Getting Started
```

Comment:
- [OPTIONAL]  Please, add a meaningful  `Project description`. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how. 😉 💯 

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.